Performance/FlxBunnyMark/source/PlayState.hx

```diff
@@ -17,10 +18,11 @@ import flixel.util.FlxColor;
  */
 class PlayState extends FlxState
 {
-	public static inline var INITIAL_AMOUNT:Int = 1000;
+	public static inline var INITIAL_AMOUNT:Int = 100;
```

Comment:
Did you decrease this amount as a result of the shader changes / are they performance-intensive for you? 100 seems too little for a performance benchmark demo to me. I think this also impacts the amount of bunnies added with each button click.